### ISSUE TITLE
Ensure Tooling is Up-to-Date

### ISSUE THREAD

**Issue by HCastano** [paritytech/canvas#98]
Before launch, we should make sure that tools like [Sidecar](https://github.com/paritytech/substrate-api-sidecar) and [TxWrapper](https://github.com/paritytech/txwrapper-core/tree/main/packages/txwrapper-polkadot) are able to support Canvas. This will make chain explorer and exchange integration easier.

**Comment by cmichi** [comment]
@dvdplm Do you have an idea if there are changes necessary in those components to support a Canvas parachain?

**Comment by dvdplm** [comment]
Yes! :) So [this is a guide](https://github.com/paritytech/substrate-api-sidecar/blob/master/guides/CHAIN_INTEGRATION.md) to help integrators build out support for their chains for inclusion in `sidecar`. You can see examples of how others do it [here](https://github.com/paritytech/substrate-api-sidecar/tree/master/src/chains-config) (Acala, Astar, Kilt, Calamari etc). For `txwrapper-core` the equivalent guide is [here](https://github.com/paritytech/txwrapper-core/blob/main/CHAIN_BUILDER.md).

We're very happy to help out too ofc!

/cc @TarikGul @jarcodallo

**Comment by TarikGul** [comment]
We have also discussed creating a `/contracts` endpoint to better facilitate any core logic that is not yet implemented in sidecar. Similar [issue](https://github.com/paritytech/substrate-api-sidecar/issues/795).

@cmichi Curious what your thoughts are on it, and if there is anything you see that should be supported by default.
